onnxruntime/test/onnx/main.cc

```diff
@@ -299,8 +299,6 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
       {"BatchNorm2d_momentum_eval", "disable reason"},
       {"BatchNorm3d_eval", "disable reason"},
       {"BatchNorm3d_momentum_eval", "disable reason"},
-      {"constantofshape_float_ones", "test data bug"},
```

Comment:
Since [this](https://github.com/onnx/onnx/issues/1842) issue still persists, aren't these tests still expected to fail [here](https://github.com/microsoft/onnxruntime/blob/fd5eb3b351780e34893c7c3ccf2319db1846e58d/onnxruntime/core/providers/cpu/generator/constant_of_shape.cc#L167) ?

Reply:
They do not fail. I think the issue was: ONNX bug. Since then we picked up the fix.

---
In reply to: [285319784](https://github.com/microsoft/onnxruntime/pull/1059#discussion_r285319784) [](ancestors = 285319784)

Reply:
This is strange. 

I have 64 bit python and when I create the shape tensor like the test case [here](https://github.com/onnx/onnx/blob/819afd05a885921dd4dea066749ca08661daefc8/onnx/backend/test/case/node/constantofshape.py#L17), and when I print out the `dtype` , it says it is of type int32 - 

![image](https://user-images.githubusercontent.com/9969784/57962118-a7789280-78c8-11e9-9ab9-3d9cd4e43a97.png)

This should result in runtime failure as it violates the spec. It seems to pass CI which means the shape tensor seems to be of type int64.

Reply:
You are on windows, right? Qualify the type of the array with the type you want

Reply:
This is addressed by fixing the model on the tip of onnx we use. The older releases will have this test disabled.

---
In reply to: [285320751](https://github.com/microsoft/onnxruntime/pull/1059#discussion_r285320751) [](ancestors = 285320751)